testdata/p4_16_samples_outputs/table-entries-range-bmv2.p4.entries.txt

```diff
@@ -0,0 +1,75 @@
+updates {
+  type: INSERT
+  entity {
+    table_entry {
+      table_id: 33595117
+      match {
+        field_id: 1
+        range {
+          low: "\001"
+          high: "\010"
+        }
+      }
+      action {
+        action {
+          action_id: 16837978
+          params {
+            param_id: 1
+            value: "\000\025"
+          }
+        }
+      }
+      priority: 1
+    }
+  }
+}
+updates {
+  type: INSERT
+  entity {
+    table_entry {
+      table_id: 33595117
+      match {
+        field_id: 1
+        range {
+          low: "\006"
+          high: "\014"
+        }
+      }
+      action {
+        action {
+          action_id: 16837978
+          params {
+            param_id: 1
+            value: "\000\026"
+          }
+        }
+      }
+      priority: 2
+    }
+  }
+}
+updates {
+  type: INSERT
+  entity {
+    table_entry {
+      table_id: 33595117
+      match {
+        field_id: 1
+        range {
+          low: "\000"
+          high: "\377"
```

Comment:
Same comment here about "don't care" fields, whether they should be omitted or present in the file.